mmv1/templates/terraform/custom_import/biglake_database.go.erb

```diff
@@ -0,0 +1,18 @@
+    config := meta.(*transport_tpg.Config)
```

Comment:
We may not need a custom import if we use {{% in the import & ID format?

See resources like: https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/products/healthcare/ConsentStore.yaml

Reply:
Done & fixed a couple other issues with the original PR as well.